core/tests/vm-benchmark/src/vm.rs

```diff
@@ -282,4 +292,22 @@ mod tests {
         let res = vm.run_transaction(&get_heavy_load_test_tx(1));
         assert_matches!(res.result, ExecutionResult::Success { .. });
     }
+
+    #[test]
+    fn instruction_count_matches_on_both_vms_for_transfer() {
+        let tx = get_transfer_tx(0);
+        let legacy_count = Legacy::count_instructions(&tx);
+        let fast_count = Fast::count_instructions(&tx);
+        assert_eq!(legacy_count, fast_count);
+    }
+
+    #[test]
+    fn instruction_count_matches_on_both_vms_for_fuzzed_bytecodes() {
```

Comment:
confusing naming: I think BYTECODES is the benchmark bytecodes, not the fuzzing corpus.